### ISSUE TITLE
Add `hgv` to access field

### ISSUE THREAD

**Issue by arch0345** [openstreetmap/id-tagging-schema#526]
Ideally this should only be visible for highway types where motor vehicles are allowed (ie not footways). Adding [`hgv`](https://wiki.openstreetmap.org/wiki/Key:hgv) would make it easier for mappers to indicate whether or not trucks are allowed and whether through truck traffic is allowed.

Truck access signage is relatively common and can be mapped by looking at street-level imagery. For example:

![Screenshot 2022-07-11 135944](https://user-images.githubusercontent.com/85302075/178359024-653a983d-1d01-4a3a-8136-1272eba24c12.png)
`hgv=destination`

<img src="https://live.staticflickr.com/7152/6549496903_a825cf8400_n.jpg" width="320" height="240" alt="No Trucks Allowed Traffic Sign"></a>
`hgv=no`

**Comment by 1ec5** [comment]
Previously openstreetmap/iD#3946.

**Comment by tyrasd** [comment]
also related: https://github.com/openstreetmap/iD/issues/3945 and #378

**Comment by RudyTheDev** [comment]
Anecdotally, consider that `hgv` is used 770K times and `horse` is used 1.5M times. I have to wonder how much of it is due to iD having the latter field, but not the former (and also it being a weird acronym). Given that `access` is de jure for legal restriction but a de facto mix of legal and intended/implied/possible use. Personally, I know of a single location in my area that has an actual legal `horse=no` traffic signed area, and I have not seen a single other use of `horse` that was tagged for any other reason than "I think horses can('t) go here". Yet every tenth road<sup>[citation needed]</sup> has a "no hgv" sign -- these are basically all over the place. If anything, I would get rid of the `horse` field than not have an `hgv` field. Ideally though, `horse` should be only shown in countries/regions where it's actually commonplace to have traffic signs for horses and bridleways and such, like UK.

Basically, this sign is everywhere in Europe:

![Zeichen_253_-_Verbot_für_Kraftfahrzeuge_mit_einem_zulässigen_Gesamtgewicht,_StVO_1992 svg](https://user-images.githubusercontent.com/3857299/194870277-ac261748-d978-48a0-9eea-046bbc1b0681.png)

**Comment by AntMadeira** [comment]
This would be a great addition. And imho it even could substitute the horse access field.

**Comment by matkoniecz** [comment]
Maybe it would be possible to design in way that would make it appear if tagged? Even if not shown all the time.

On other hand it seems important enough to show all the time and definitely more important than horse=
